apps/src/lab2/constants.ts

```diff
@@ -27,3 +27,6 @@ export enum PERMISSIONS {
 export const START_SOURCES = 'start_sources';
 
 export const LABS_USING_NEW_SHARE_DIALOG = ['music', 'pythonlab'];
+
+// Text-based labs that are currently supported by lab2.
```

Comment:
Does "text-based" here mean the student source is created in a text editor?

Reply:
Kind of, yes - this refers to any lab where student source is stored as text. Sometimes this is in a text-based IDE (python, weblab) but not always, for example in AI Chat.

I believe this is primarily used to determine what string to display in the start over message (referring to "remove all blocks" vs "remove all code")